allan_variance.py

```diff
@@ -120,3 +129,23 @@ def params_from_avar(tau, avar):
     prediction = A.dot(x) * avar
 
     return np.sqrt(x), prediction
+
+
+def allan_deviation(avar):
```

Comment:
I think it's not worth it to keep a separate function which just computes sqrt. I suggest to stick with `allan_variance` and that's it.